kickstarter/android-oss
# NT-4123: Addon card component with fixed missing description

# 📲 What

- Updated this to fix the broken description disappearing on addons that should have it

# 🛠 How

<img width="576" alt="Screen Shot 2021-08-19 at 10 26 55 AM" src="https://user-images.githubusercontent.com/19390326/130086394-65fc985f-f9d3-45c2-9e91-1c0667e3d668.png">

# 👀 See

<img src="https://user-images.githubusercontent.com/19390326/130086936-47ba2a01-f21d-468e-8323-88855481c723.png" height="600">

# 📋 QA

- Go to the project GURLZ - An artbook by Serge Birault
- Navigate to project addons
- Description should be visible

# Story 📖

[NT-2143: Refactor the AddOns screen to use the re-usable components](https://kickstarter.atlassian.net/browse/NT-2143)
